cypress/page/login.page.ts

```diff
@@ -0,0 +1,29 @@
+class LoginPage {
+    private username: string;
+
+    private userPos: string;
+
+    private password: string;
+
+    private passPos: string;
+
+    private submitInfo: string;
+
+    constructor() {
+      this.userPos = "#email";
+      this.username = "aperdomobo@gmail.com";
+      this.password = "WorkshopProtractor";
+      this.passPos = "#passwd";
+      this.submitInfo = "#SubmitLogin";
+    }
+
+    public fillInfo(): void {
+      cy.get(this.userPos).type(this.username);
+      cy.get(this.passPos).type(this.password);
+    }
+
+    public submitLoginInfo(): void {
+      cy.get(this.submitInfo).click();
+    }
```

Comment:
These 2 methods could be just one called login, at that method should receive user and pass as parameters, in that way, the loginPage object will be reusable for any user and pass